wtransport/Cargo.toml

```diff
@@ -35,7 +35,7 @@ rustls-pemfile = "1.0.2"
 socket2 = "0.5.3"
 thiserror = "1.0.40"
 time = { version = "0.3.21", optional = true }
-tokio = { version = "1.28.1", default-features = false, features = ["macros", "fs"] }
+tokio = { version = "1.28.1", default-features = false, features = ["macros", "fs", "rt-multi-thread"] }
```

Comment:
Why do we need this?

Those are library dependency (not `[dev-dependencies`]).

`wtransport` should *not* need `rt-multi-thread` as it does not start any runtime itself. The runtime is only used as `dev-dependencies` because the example.

Reply:
Oh, when I tried to compile full.rs, it gave me an error that I have to enable this feature. When I enabled, it compiles successfully.

Reply:
That's interesting. 

What are the exact commands to reproduce the issue?

Reply:
So are you able to reproduce if you perform the following steps:

```
git clone https://github.com/BiagioFesta/wtransport.git && cd wtransport/ && cargo run --example full
```

?

Reply:
I think I CD into the examples directory first, then just run the cargo run command 😄

Reply:
I see, nice finding!

Reply:
Did you reproduce it? Or I can confirm if you give me few minutes

Reply:
I can confirm the issue. Thanks!